clang/bindings/python/clang/cindex.py

```diff
@@ -1763,19 +1799,22 @@ def get_usr(self):
         another translation unit."""
         return _CXString.from_result(conf.lib.clang_getCursorUSR(self))
 
-    def get_included_file(self):
+    @cursor_null_guard
+    def get_included_file(self) -> File:
         """Returns the File that is included by the current inclusion cursor."""
         assert self.kind == CursorKind.INCLUSION_DIRECTIVE
 
         return File.from_result(conf.lib.clang_getIncludedFile(self), self)
 
     @property
-    def kind(self):
+    @cursor_null_guard
```

Comment:
You replaced `@property` decorator with your new decorator. Isn't that an unnecessary API break? Combining decorators is perfectly fine if done in the correct order.

Can you ensure we have a test for `kind` working as a property?

Reply:
Nevermind, I misread the diff as if you replaced `@property` with `@cursor_null_guard`, but now I see that you preserved the original decorator.